guides/v2.2/cloud/project/new-relic.md

```diff
@@ -57,11 +68,13 @@ runtime:
         - newrelic
 ```
 
-## Manage license keys
+## Connect to New Relic
+
+To connect a Cloud environment to New Relic, you must add the New Relic license key to the environment.
 
-For Pro accounts, there is no need to apply the license key to any environments. This has already been done. You can log in to your New Relic account and verify connectivity.
+-  For Pro plan accounts, Magento adds the license key to your Cloud environments during the provisioning process. You can verify the connection by logging into your New Relic account.
```

Comment:
You can log in to New Relic, but that doesn't necessarily verify connectivity to your Cloud.